pkg/auth/providerrefresh/refresher.go

```diff
@@ -47,6 +48,7 @@ type refresher struct {
 	intervalInSeconds   int64
 	unparsedMaxAge      string
 	maxAge              time.Duration
+	cron                cron.Cron
```

Comment:
I'd make this `*cron.Cron` which would let you use the return value from `cron.New()`?